### PR TITLE
Cron contract updates

### PR DESCRIPTION
**Pull request by begmaroman**

### CODE REVIEW

File: contracts/src/v0.8/upkeeps/CronUpkeep.sol [hunk 1]
```diff
@@ -103,26 +106,16 @@ contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pa
    * @param id the id of the cron job to delete
    */
   function deleteCronJob(uint256 id) external onlyOwner {
```
Comment (RyanRHall) [inline contracts/src/v0.8/upkeeps/CronUpkeep.sol:108]: can we add a test that tests the revert?

image

Comment (RyanRHall) [reply contracts/src/v0.8/upkeeps/CronUpkeep.sol:108]: also, can we move the if ... revert.. into a modifier? like `onlyValidCronID`  or something similar? we can use it here and in `getCronJob`

File: contracts/src/v0.8/upkeeps/CronUpkeep.sol [hunk 2]
```diff
@@ -175,6 +173,10 @@ contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pa
       uint256 nextTick
     )
   {
+    if (!s_activeCronJobIDs.contains(id)) {
```
Comment (RyanRHall) [inline contracts/src/v0.8/upkeeps/CronUpkeep.sol:176]: can we add a test for this reversion as well?

File: contracts/test/v0.8/CronUpkeep.test.ts [hunk 3]
```diff
@@ -349,10 +349,7 @@ describe('CronUpkeep', () => {
       await createBasicCron()
       await assertJobIDsEqual([1, 2, 3, 4])
       await cron.deleteCronJob(2)
-      assert.equal(
-        (await cron.getCronJob(2)).target,
-        ethers.constants.AddressZero,
-      )
+      await expect(cron.getCronJob(2)).to.be.revertedWith(CRON_NOT_FOUNR_ERR)
```
Comment (RyanRHall) [inline contracts/test/v0.8/CronUpkeep.test.ts:352]: CRON_NOT_FOUNR_ERR --> CRON_NOT_FOUND_ERR

File: contracts/src/v0.8/upkeeps/CronUpkeep.sol (hunk 2 above)
Comment (begmaroman) [reply contracts/src/v0.8/upkeeps/CronUpkeep.sol:176]: We already have a test for this on 352 line.

File: contracts/src/v0.8/upkeeps/CronUpkeep.sol (hunk 1 above)
Comment (begmaroman) [reply contracts/src/v0.8/upkeeps/CronUpkeep.sol]: done

**Pull request merged**